rero_ils/modules/api.py

```diff
@@ -90,6 +90,16 @@ def flush_and_refresh(cls):
         """Flush and refresh index."""
         current_search.flush_and_refresh(cls.Meta.index)
 
+    def get_record_by_pid(self, pid, fields=None):
+        """Search by pid."""
+        query = self.filter('term', pid=pid).extra(size=1)
+        if fields:
+            query = query.source(includes=fields)
+        response = query.execute()
+        if response.hits.total.value != 1:
```

Comment:
What if they are two or more hits? Should the message be something like " :scream: "?

Reply:
A pid value is unique. Either it exists or you have an exception?